src/transformers/models/gpt_sw3/tokenization_gpt_sw3.py

```diff
@@ -0,0 +1,306 @@
+import os
+import re
+import unicodedata
+
+from ... import is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+from shutil import copyfile
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import sentencepiece as spm
```

Comment:
This tokenizer requires sentencepiece which is not a core dependency. Make sure the imports of the tokenizer are protected both in the main init and in the gpt_sw3 init.